<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>442</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Альфа штрих</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Создать набор точек</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Расстояниеe между точками</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dist">
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 1px solid #ef3124;</string>
     </property>
     <property name="inputMask">
      <string extracomment="Hello world"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="gen_dots_but">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>13</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #ef3124;
color: #ffffff;
border: none;
font: Roboto bold;
}
QPushButton:pressed {
	border: 1px solid #ef3124;
	color: #ef3124;
   	background-color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Построить график</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Функция</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="function">
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 1px solid #ef3124;</string>
     </property>
     <property name="inputMask">
      <string extracomment="Hello world"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Отрезок построения от</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="begin">
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 1px solid #ef3124;</string>
     </property>
     <property name="inputMask">
      <string extracomment="Hello world"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>до</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="end">
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 1px solid #ef3124;</string>
     </property>
     <property name="inputMask">
      <string extracomment="Hello world"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="upload_dots">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>207</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>13</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #ef3124;
color: #ffffff;
border: none;
font: Roboto bold;
}
QPushButton:pressed {
	border: 1px solid #ef3124;
	color: #ef3124;
   	background-color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Загрузить точки</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>177</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Вычислить интеграл</string>
    </property>
   </widget>
   <widget class="QLabel" name="path_to_file">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>214</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
border: none;
border-bottom: 1px solid #ef3124;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="left_rec_val">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="right_rec_val">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="avg_rec_val">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="trap_val">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="simpson_val">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_dots">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>291</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>13</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #ef3124;
color: #ffffff;
border: none;
font: Roboto bold;
}
QPushButton:pressed {
	border: 1px solid #ef3124;
	color: #ef3124;
   	background-color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Сохранить точки первообразной</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="avg_rec_but">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Средние прямоугольники</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="right_rec_but">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Правые прямоугольники</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="left_rec_but">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Левые прямоугольники</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="trap_but">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод трапеций</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="simpson_but">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод Симпсона</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="source_graph" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="primitive_graph" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="gen_dots_but_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>13</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #ef3124;
color: #ffffff;
border: none;
font: Roboto bold;
}
QPushButton:pressed {
	border: 1px solid #ef3124;
	color: #ef3124;
   	background-color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Сохранить точки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_source">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>13</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #ef3124;
color: #ffffff;
border: none;
font: Roboto bold;
}
QPushButton:pressed {
	border: 1px solid #ef3124;
	color: #ef3124;
   	background-color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Сбросить масштаб</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_primitive">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>400</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>13</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #ef3124;
color: #ffffff;
border: none;
font: Roboto bold;
}
QPushButton:pressed {
	border: 1px solid #ef3124;
	color: #ef3124;
   	background-color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Сбросить масштаб</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>График исходной функции</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>График первообразной</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
